src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -432,8 +450,42 @@ class WorkItemListPage {
   clickWorkItemKebabMoveToBottomButton (parentElement) {
     return this.workItemKebabMoveToBottomButton (parentElement).click();
   }
-
-
+  /**Iterations Page object model */
+  iterationAddButton  (){
+    return element(by.css('.pficon-add-circle-o.fa-icon'));
+  }
+  clickIterationCreateLabel  (){
+    return element(by.id("add-iteration")).click();
+  }
+  get iterationTitle  (){
+    return element(by.id("iteration-name")).click();
+  }
+  setIterationTitle  (newTitleString,append){
+    if (!append) { this.iterationTitle.clear(newTitleString) };
+    return this.iterationTitle.sendKeys(newTitleString);
+  }
+  get iterationDescription  (){
+    return element(by.id("iteration-description")).click();
+  }
+  setIterationDescription  (newString,append){
+     if (!append) { this.iterationDescription.clear(newString) };
+    return this.iterationDescription.sendKeys(newString);
+  }
+  clickCreateIteration  (){
```

Comment:
I  did not make separate functions to define button and directly used them, bcuz I  couldn't think of any other use case rather then clicking on it for now :). And yes we can use the general method for checking the presence of the elements. 

I will add general definition in different methods.